### PR TITLE
add vignette, update metab_simple

### PR DESCRIPTION
**Pull request by aappling-usgs**
@jread-usgs - here's your vignette! pulls data from SB and uses it to model metabolism (poorly, of course). i've git-ignored the html output of the vignette to avoid overloading the repo - let me know if you'd prefer to keep those versioned.

@robohall - i've switched metab_simple() back to computing the DO deficit from DO.sat and DO.mod[i-1], as i think you had in your example code. there are some bugs/data issues to fix, but we're on our way to having that simple model plumbed into the pipeline. talk to you tomorrow.

### CODE REVIEW

**Comment by jread-usgs** [comment]
We'll need an `install_github` for `mda.streams` and `powstreams` here: https://github.com/USGS-R/streamMetabolizer/blob/master/appveyor.yml#L11

I think it can all be done on one line like this:
```
- travis-tool.sh install_github appling/unitted@v0.2.0 USGS-R/mda.streams USGS-R/powstreams
```
Same kind of thing needed for `.travis` when we add a new dependency that isn't on CRAN. 
Want to take a swing at it?

File: vignettes/beaty_creek.Rmd [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+kable(head(beaty_creek))
+```
+
+There are nearly six years of data, with pronounced seasonal patterns in DO and temperature.
+```{r}
+beaty_creek_tidy <- beaty_creek %>%
+  rename(DO_mgL=ts_doobs, WaterTemp_C=ts_wtr, Discharge_cfs=ts_disch) %>%
+  gather(variable, value, DO_mgL, WaterTemp_C, Discharge_cfs)  
+# Plot to see data range and multi-year patterns
+ggplot(beaty_creek_tidy, aes(x=DateTime, y=value)) + geom_point(size=0.5) + facet_grid(variable~., scales="free_y") + theme_bw()
```
Comment (jread-usgs) [inline vignettes/beaty_creek.Rmd]: can we log-y on discharge in the future?

Comment (aappling-usgs) [inline vignettes/beaty_creek.Rmd]: sure. i was being lazy.

**Comment by aappling-usgs** [comment]
ok, i think the problem now is that the Travis CI build is using a different version of powstreams from mine. i'll go get that fixed now, then come back here.

**Pull request merged**